### PR TITLE
Update to Bevy 0.15

### PR DESCRIPTION
**Pull request by ptsd**
- Updates `bevy` to `0.15`
- Slightly tidies up the 2d example
- Adds the 2d example to `Cargo.toml`

### CODE REVIEW

**Comment by ptsd** [comment]
Seems there are two almost identical PRs 😄

File: Cargo.toml [hunk 1]
```diff
@@ -33,3 +34,7 @@ init = []          # Enables automatically choosing a camera
 [[example]]
 name = "3d_scene"
 path = "examples/3d_scene.rs"
+
+[[example]]
```
Comment (Shatur) [inline Cargo.toml:38]: I wondering why the original author included the `3d_scene` example in Cargo.toml. They are discovered automatically anyway.

**Comment by Shatur** [comment]
Merged #15 since I planning to remove example mentions from Cargo.toml and similar comments from the 3D example :sweat_smile:

File: Cargo.toml (hunk 1 above)
Comment (ptsd) [reply Cargo.toml:38]: Generally useful if you want to enable optional features per example, but yeah redundant in this case as there are no features. Was just following the original maintainer 🤷 😄

**Pull request closed without merging**